### ISSUE TITLE
Issue with upload

### ISSUE THREAD

**Issue by M5oul** [YunoHost-Apps/bozon_ynh#2]
I have strange thing when I upload a file, I can't do more things, it says:

``` fish
GD n'est pas installé sur ce serveur En savoir plus
Curl n'est pas installé sur ce serveur En savoir plus
```

`curl` is installed.

**Comment by ewilly** [comment]
hum est-ce que php5-curl & php5-gd sont installés ?

**Comment by M5oul** [comment]
Nope, I installed them and it works fine.

``` fish
sudo apt-get install php5-curl php5-gd
```

After you add this line, I could add your app in community apps.

**Comment by ewilly** [comment]
Fixed in revision 7602e9b19df25d4b11e7827e3dc4a6f616731e18
